src/styles/elements/_table.css

```diff
@@ -9,7 +9,7 @@ th {
   font-size: 14px;
   line-height: 22px;
 
-  vertical-align: initial;
+  vertical-align: middle;
```

Comment:
the buttons are slightly higher than the line height and the row content was slightly uneven between the cells

Reply:
This has an effect on the label in mobile table header.